### DISCUSSION TITLE
Alarm input

### DISCUSSION THREAD

**Discussion by jtorresIndusta** [phil-barrett/grblHAL-teensy-4.x#88] [Q&A] [answered]
After using one of Phil's boards for a while I just bought the second one. I am planning on using closed loop stepper motors on this one. I assume that at this point the encoder circuit will only be meaningful to the stepper driver and not to the controller but, is there a way to connect the alarm output that this type of drivers have to the controller? In that case, which is the correct input to do so? It would be great being able to at least stop or pause the job if one of the drivers detect a position error.

**Comment by phil-barrett** [comment, marked as answer]
Most people connect alarm to Halt/Reset input. Usually the alarm output is open collector so you can connect multiple servo alarm outputs to it. Any other switches connected to the Halt input must be NO switches or open collector.
